### PR TITLE
Add CoreFx PreReleaseGroup to baseline package

### PR DESCRIPTION
**Pull request by ericstj**
This enables folks using the baseline package to get the right version information for CoreFx packages.

/cc @StephenBonikowsky @weshaggard

### CODE REVIEW

**Comment by ericstj** [comment]
I expect this to fail until I have an updated buildtools.

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 1]
```diff
@@ -11,84 +11,19 @@
...
-        "5.0.2"
-      ]
-    },
     "Microsoft.NETCore.Platforms": {
       "StableVersions": [
         "1.0.0",
         "1.0.1"
       ]
     },
-    "Microsoft.NETCore.Portable.Compatibility": {
```
Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: Are these removals unrelated?

Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: actually these look like the non-corefx packages. Where did you move them to?

**Comment by weshaggard** [comment]
LGTM

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json (hunk 1 above)
Comment (ericstj) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: For WCF ones they are already defined in the WCF index.

None of these should ever be used by corefx, its at the bottom of the stack.  Its a problem (repo cycle) if we start depending on them.  I'd expect these to be defined in the repo where the packages live (coreclr, core-setup, or WCF).

**Comment by ericstj** [comment]
@dotnet-bot test this please

**Comment by ericstj** [comment]
@dotnet-bot test this please

**Comment by ericstj** [comment]
@dotnet-bot test this please

**Pull request merged**